src/main/java/org/opentripplanner/util/OTPFeature.java

```diff
@@ -0,0 +1,118 @@
+package org.opentripplanner.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * The purpose of this class is to be able to turn features on and off.
+ * <p/>
+ * This configuration is optional an found under "feature" in the top
+ * level 'otp-config.json' file.
+ */
+public enum OTPFeature {
+    APIExternalGeocoder(true),
+    APIBikeRental(true),
+    APIAlertPatcher(true),
+    APIRouters(true),
+    APIServerInfo(true),
+    APIGraphInspectorTile(true),
+    APIUpdaterStatus(true);
+
+    private static final String VALUE_ON = "on";
+    private static final String VALUE_OFF = "off";
+    private static final String VALUE_ENABLE = "enable";
+    private static final String VALUE_DISABLE = "disable";
+
+    private static final Logger LOG = LoggerFactory.getLogger(OTPFeature.class);
+
+    OTPFeature(boolean defaultEnabled) {
+        this.enabled = defaultEnabled;
+    }
+
+    private boolean enabled;
+
+    /**
+     * Return {@code true} if feature is turned 'on'.
+     */
+    public boolean on() {
```

Comment:
By convention we should call this “isOn” or “isEnabled”.

Reply:
`isOn` is chosen